core/internal/gethwrappers/generation/generated-wrapper-dependency-versions-do-not-edit.txt

```diff
@@ -1,7 +1,7 @@
-GETH_VERSION: 1.9.12
 flux_aggregator_wrapper: ../../../evm-contracts/abi/v0.6/FluxAggregator.json 3b870259faea720a909bf4be02e0045df905395a986576f2473aa3910834b73d
+GETH_VERSION: 1.9.12
```

Comment:
Why has this moved?

Reply:
It's autogenerated. Not sure what the generator did there